src/main/java/io/reactivex/rxjava3/internal/schedulers/IoScheduler.java

```diff
@@ -212,11 +218,20 @@ public void dispose() {
             if (once.compareAndSet(false, true)) {
                 tasks.dispose();
 
-                // releasing the pool should be the last action
-                pool.release(threadWorker);
+                if (USE_SCHEDULED_RELEASE) {
+                    threadWorker.scheduleActual(this, 0, TimeUnit.NANOSECONDS, null);
```

Comment:
I'm not sure if this is preferred to `threadWorker.scheduleDirect(this, 0, TimeUnit.NANOSECONDS);`
Any comment about this?

Reply:
Doesn't really matter.